observations/obs_converters/AIRS/amsua_bt_mod.f90

```diff
@@ -0,0 +1,1100 @@
+
+module amsua_bt_mod
+
+! the contents of this file are a consolidation of:
+!    amsua_bt_typ.inc
+!    amsua_bt_struct.inc
+! modified to use fortran 90, removed redundant type declarations
```

Comment:
These original files came from NASA and were really unneccessarily complex F77 files.
I have modified these substantially, probably enough to put the DART copyright on them.  Thoughts?